targets/l2_switch/Makefile

```diff
@@ -52,6 +52,7 @@ bm-p4ofagent : BM_PARAMS += --plugin of
 bm-p4ofagent : BM_PARAMS += --openflow-mapping-dir $(TARGET_ROOT)/openflow_mapping
 bm-p4ofagent : BM_PARAMS += --openflow-mapping-mod l2
 bm-p4ofagent : export LIB_P4OFAGENT_ENABLE=1
+bm-p4ofagent : export PLUGIN_OPENFLOW=1
```

Comment:
I believe this won't produce the desired behavior, because target-dependent code is executed on the "second pass" after all the includes and rules have been built. To include the openflow-plugin.mk makefile in p4c-behavioral/p4c_bm/plugin, PLUGIN_OPENFLOW must be defined in the "first pass." So people should build targets like this: 

make bm-p4ofagent PLUGIN_OPENFLOW=1